torch/nn/modules/pooling.py

```diff
@@ -601,7 +601,7 @@ class AvgPool2d(_AvgPoolNd):
     count_include_pad: bool
 
     def __init__(self, kernel_size: _size_2_t, stride: Optional[_size_2_t] = None, padding: _size_2_t = 0,
-                 ceil_mode: bool = False, count_include_pad: bool = True, divisor_override: bool = None) -> None:
+                 ceil_mode: bool = False, count_include_pad: bool = True, divisor_override: int = None) -> None:
```

Comment:
Hmm, shouldn't it be `Optional[int]`, shouldn't it?

Reply:
yes indeed. @guilhermeleobas can you update?

Reply:
Done!